Gopkg.lock

```diff
@@ -1388,6 +1422,14 @@
   revision = "bb7a9ca6e4fbc2129e3db588a34bc970ffe811a9"
   version = "v2.5.1"
 
+[[projects]]
+  digest = "1:df89444601379b2e1ee82bf8e6b72af9901cbeed4b469fa380a519c89c339310"
+  name = "gopkg.in/logfmt.v0"
```

Comment:
Let me try requesting this on their github page. In general, I'd imagine this would be a recurring problem (since gopkg.in exists and many developers prefer using it for versioning). Things only get worse if such libs are pulled in as transitive dependencies (e.g. `signalfx/golib --> github.com/foobar --> github.com/frobnicator --> gopkg.in/logfmt`) in such cases wouldn't we rather just let the pkg/module management layer handle this for us (since they are unique names anyways)?

Reply:
We are using gopkg.in for some packages but in this case the logfmt project is not suggesting it as the canonical path, and we would like to use the recommended import path.  It would work with both copies included, though it's undesirable since it just adds code needlessly to the binary.  Let's see what signalfix says.

Reply:
Link to upstream PR changing this: https://github.com/signalfx/golib/pull/134